Sources/AWSLambdaEvents/APIGateway.swift

```diff
@@ -37,6 +37,7 @@ public struct APIGatewayRequest: Codable {
 
         public let resourceId: String
         public let apiId: String
+        public let domainName: String
```

Comment:
should this be optional?

Reply:
I have yet to find a request that doesn't include this field, but I'm happy to make it optional if that would reduce potential breakages under corner cases I haven't seen yet.

Reply:
Done